azext_iot/central/models/v1_1_preview/device_group.py

```diff
@@ -10,3 +10,5 @@ def __init__(self, group: dict):
         self.display_name = group.get("displayName")
         self.id = group.get("id")
         self.organizations = group.get("organizations")
+        self.filter = group.get("filter")
```

Comment:
@lucadruda  I created ga_2022_05_31 folder for this model.

Reply:
can etag be provided by the user when run update commands? if yes we can add it to the model, otherwise this will simply be ignored in the output

Reply:
No, it can only be specified in create-device-group command. We don't need to add this into model but we can accept etag when user create device group.